easybuild/easyconfigs/s/spglib/spglib-1.9.4.2-intel-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = "PythonPackage"
+
+name = 'spglib'
```

Comment:
@boegel regarding the urls, I asked because in this case `pypi.python.org` is failing... (I'm using v2.8.2, so already with that change)

regarding the module name, ok

regarding `--update-pr`, cool, will have a look at it